### DISCUSSION TITLE
Boxart with Subfolders

### DISCUSSION THREAD

**Discussion by dh3lix** [OnionUI/Onion#620] [Q&A] [answered]
Hey everybody,

I am trying to get Boxarts working with subfolders .. eg this is the current romset in MD ..

![image](https://user-images.githubusercontent.com/1379420/205859494-d41fd864-a203-4a3e-8ff9-6105cb9e6530.png)

So all the rom images are in those subfolders. Scraping accoringly will result in Imgs in main folder and the gamelist.xml in the root of the MD - the paths are relative correct in the XML. Boxarts will not be shown in the Onion frontend tho. I also tried editing the XML to fit absolute paths with no success tho. Any idea or simply does not work that way in Onion?

Have a great Tuesday and stay safe!
marcO

**Comment by Aemiii91** [comment, marked as answer]
The xml file needs to be named `miyoogamelist.xml` to be recognized. Usually the Imgs folder should have an entirely flat structure.
